### PR TITLE
Increase audit log size

### PR DESCRIPTION
**Pull request by eric3873**
Add test

### CODE REVIEW

**Review by jeffdegott — APPROVED** [review]

**Comment by jeffdegott** [comment]
waiting for green light....

File: src/fty-alert-engine-log.cfg [hunk 1]
```diff
@@ -20,8 +20,8 @@ log4cplus.logger.alerts-engine-audit=DEBUG, alerts-engine-audit-log, syslog, con
 
 log4cplus.appender.alerts-engine-audit-log=log4cplus::RollingFileAppender
 log4cplus.appender.alerts-engine-audit-log.File=/var/log/app-audit/alarms-audit.log
-log4cplus.appender.alerts-engine-audit-log.MaxFileSize=16MB
-log4cplus.appender.alerts-engine-audit-log.MaxBackupIndex=3
+log4cplus.appender.alerts-engine-audit-log.MaxFileSize=64MB
+log4cplus.appender.alerts-engine-audit-log.MaxBackupIndex=10
 log4cplus.appender.alerts-engine-audit-log.Threshold=INFO
 log4cplus.appender.alerts-engine-audit-log.layout=log4cplus::PatternLayout
 log4cplus.appender.alerts-engine-audit-log.layout.ConversionPattern=%D{%Y-%m-%d %H:%M:%S} %c [%-5p] %F:%L %m%n
```
Comment (NavarroPierre) [inline src/fty-alert-engine-log.cfg:27]: %F:%L this should not be probably in audit file.

File: src/fty-alert-engine-log.cfg [hunk 2]
```diff
@@ -20,8 +20,8 @@ log4cplus.logger.alerts-engine-audit=DEBUG, alerts-engine-audit-log, syslog, con
```
Comment (NavarroPierre) [inline src/fty-alert-engine-log.cfg:20]: Do we need to add append property to true or is it by default active.
We should not erase file at restart of the service

Comment (eric3873) [reply src/fty-alert-engine-log.cfg:20]: Without option, it is the default behavior (append)

File: src/fty-alert-engine-log.cfg (hunk 1 above)
Comment (eric3873) [reply src/fty-alert-engine-log.cfg]: Ok take into account

Comment (eric3873) [reply src/fty-alert-engine-log.cfg]: Ok take into account

**Review by NavarroPierre — DISMISSED** [review]

File: src/fty-alert-engine-log.cfg (hunk 1 above)
Comment (eric3873) [reply src/fty-alert-engine-log.cfg]: Take into account

File: src/fty-alert-engine-log.cfg (hunk 2 above)
Comment (eric3873) [reply src/fty-alert-engine-log.cfg:20]: done active by default

**Review by perrettecl — APPROVED** [review]

**Pull request merged**